### ISSUE TITLE
User not asked for credentials

### ISSUE THREAD

**Issue by bragagia** [Scalingo/cli#283]
The first three time I try to use the CLI, i'm getting an error saying I'm not authorized without any request of credentials.
```
kanak$ scalingo user-timeline
 !     An error occured:
       unauthorized - you are not authorized to do this operation
kanak$  scalingo user-timeline
 !     An error occured:
       unauthorized - you are not authorized to do this operation
kanak$ scalingo apps        
 !     An error occured:
       unauthorized - you are not authorized to do this operation
kanak$ scalingo user-timeline
-----> Currently unauthenticated
       Trying login with SSH…
 !     SSH connection failed.
       Trying login with user/password:

       Username or email: ...
```
Having a ssh key on my computer, but not configured with my Scalingo account.

**Comment by bragagia** [comment]
I'm unable to reproduce this bug and it append on local dev stack so i close this issue.
Please add a comment if you are concerned.

**Comment by bragagia** [comment]
```bash
~ scalingo --app myapp stats                                                                                                                       129 ↵
-----> Currently unauthenticated
       Trying login with SSH…
-----> Hello bragagia, nice to see you!
 !     SSH connection failed.
       Trying login with user/password:

       Username or email:
```

Seems linked.

Cli auth infos are empty :
```bash
~ cat .config/scalingo/auth
~ 
```

ssh correctly works : 
```bash
~ ssh -T git@scalingo.com
You've successfully authenticated on Scalingo, but there is no shell access
~   
```

My key is correctly registered in ssh-agent : 
```bash
~ ssh-add -l
2048 SHA256:xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx /home/kanak/.ssh/id_rsa (RSA)
~ 
```

It appens on released version and on git version.

**Comment by bragagia** [comment]
Solved by #293
